tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -49,6 +49,10 @@ pub(crate) struct TestCommand {
     /// JSON RPC url to solve oracle calls
     #[clap(long)]
     oracle_resolver: Option<String>,
+
+    /// Flag to skip tests using oracles.
```

Comment:
```suggestion
    /// Flag to skip tests using unhandled oracles without mocks.
```
The way it's phrased sounds like any test with an oracle will be skipped.